tests/integration/helpers.py

```diff
@@ -139,6 +140,50 @@ def restart_unit(model_full_name: str, unit: str) -> None:
     )
 
 
+@retry(
+    stop=stop_after_attempt(3),
+    wait=wait_fixed(5),
+    retry_error_callback=lambda _: False,
+    retry=lambda x: x is False,
+)
+def dashboard_unavailable(host: str, https: bool = False) -> bool:
```

Comment:
Can we have `dashboard_available` instead?  Otherwise we have double negatives here...

A normal check will be in this case:
```
if not dashboard_unavailable():
    # Dashboard is healthy
```

Reply:
We already do...

It's called `access_dashboard()` and `acceess_all_dashboards()`. Used across ALL test modules to check 4 healthy status (including the one in this PR).

More info in comment https://github.com/canonical/opensearch-dashboards-operator/pull/91#discussion_r1677578612

Reply:
@juditnovak then what is this module adding here?